### DISCUSSION TITLE
Access bookmark via title

### DISCUSSION THREAD

**Discussion by toomsie** [qutebrowser/qutebrowser#7083] [Q&A]
I added two bookmarks successfully and hope to access the second one simply by typing
**:bookmark-load list**. it will not work as the first url https://www.ebay.co.uk/sl/prelist/suggest?sr=shstart has **list(prelist)** in it, so I have to use the arrow keys to select the **list** bookmark underneath.

I would have thought the command bookmark-load would ignore URLs and only except titles.  Mixing them will only create conflicts.

**Comment by The-Compiler** [comment]
It sounds like you might want to create a quickmark for `list` instead?

**Reply by toomsie** [reply]
It worked perfectly many thanks.
